content/guides/dashboard/users.md

```diff
@@ -60,3 +60,11 @@ The permissions for each user role for the Dashboard Service.
 Users can "request" access to a given organization. If a developer on your team has access to Cypress and your project's source code - they can request to be given access to your organization. This means instead of you having to invite team members up front, they can request access and you can choose to accept or deny them access.
 
 <DocsImage src="/img/dashboard/request-access-to-organization.png" alt="Request access to project" ></DocsImage>
+
+## User updates
+
+Changes can be made to both the primary email address associated with your Dashboard account and the email address for billing notifications.
+
+For updating the primary email address associated with your Dashboard, click your Profile picture in the upper left corner of the Organizations page. Select **Manage Profile**. Go to the **Email** field and select your preferred email address. _Note: The email list is limited to emails provided by login provider._
+
+If you would like to update the billing email address, that can be done via the **Billing & Usage** page within the [Dashboard](https://on.cypress.io/dashboard) or you contact them directly at [billing@cypress.io](mailto:support@cypress.io).
```

Comment:
Hey Shawn,
The email link displays `billing@cypress` but the actual mailto link is for `support@cypress`

Reply:
Good eye. This has been corrected.